dask/dataframe/core.py

```diff
@@ -548,62 +654,56 @@ def __getitem__(self, key):
                           self.name, self.divisions)
         raise NotImplementedError()
 
+    @wraps(pd.DataFrame._get_numeric_data)
+    def _get_numeric_data(self, how='any', subset=None):
+        return self
+
+    def _validate_axis(self, axis=0):
+        if axis not in (0, 'index', None):
+            raise ValueError('No axis named {0}'.format(axis))
+        # convert to numeric axis
+        return {None: 0, 'index': 0}.get(axis, axis)
+
+    def _aca_agg(self, token, func, aggfunc=None):
+        """ Wrapper for aggregations """
+        if aggfunc is None:
+            aggfunc = func
+
+        return aca([self], chunk=func,
+                   aggregate=lambda x: aggfunc(pd.Series(x)),
+                   columns=return_scalar, token=self._token_prefix + token)
+
     @wraps(pd.Series.sum)
-    def sum(self):
-        return reduction(self, pd.Series.sum, pdsum, token='series-sum')
+    def sum(self, axis=None):
+        return super(Series, self).sum(axis=axis)
```

Comment:
Why can't we rely on normal inheritance for this?  Is this just for the docstring?